v2/src/modules/address/components/EthBalanceTabs/TableTxsHistoryRow.vue

```diff
@@ -0,0 +1,315 @@
+<template>
+    <div class="position-relative">
+        <v-row v-if="smAndDown" class="my-5" justify="space-between" @click="showMoreDetails = !showMoreDetails">
+            <v-col cols="6">
+                <v-avatar :color="transferDirection.color" size="22">
+                    <v-icon size="13" color="white"> {{ transferDirection.icon }} </v-icon>
+                </v-avatar>
+                <span class="ml-2">
+                    {{ transferDirection.text }}
+                </span>
+            </v-col>
+            <v-col cols="6" class="text-right">{{ txFee.value }} ETH</v-col>
+            <v-col cols="6" class="text-info text-lowercase"> {{ timestamp }} {{ transferDirection.direction }} </v-col>
+            <v-col cols="6">
+                <div class="d-flex align-center justify-end">
+                    <app-address-blockie :address="txAddress || ''" :size="6" class="mr-2 mr-sm-2" />
+                    <app-transform-hash is-short is-blue :hash="eth.toCheckSum(txAddress)" :link="`/address/${txAddress}`" />
+                </div>
+            </v-col>
+        </v-row>
+        <v-row v-else class="my-5" align="center">
+            <v-col :sm="mdAndDown ? 3 : 2">
+                <div class="d-flex">
+                    <span style="width: 30px" class="d-inline-block">
+                        <v-icon :color="statusIcon.color">{{ statusIcon.icon }}</v-icon>
+                    </span>
+                    <div class="ml-4">
+                        {{ txFee.value }} ETH
+                        <div class="text-lowercase">
+                            {{ timestamp }} <span v-if="mdAndDown">{{ transferDirection.direction }}</span>
+                        </div>
+                    </div>
+                </div>
+            </v-col>
+            <v-col v-if="!mdAndDown" cols="2">
```

Comment:
this whole row should be clickable rather then at the button area

Reply:
Yes, that's the plan